src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.converter.FileReaderImpl;
+import core.basesyntax.converter.FileWriter;
+import core.basesyntax.converter.FileWriterImpl;
+import core.basesyntax.operation.BalanceHandler;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.OperationStrategyImpl;
+import core.basesyntax.operation.PurchaseHandler;
+import core.basesyntax.operation.ReturnHandler;
+import core.basesyntax.operation.ShopService;
+import core.basesyntax.operation.ShopServiceImpl;
+import core.basesyntax.operation.SupplyHandler;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        FileReaderImpl fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.txt");

```

Comment:
You should avoid using absolute path to a resource. Everyone who pulls your project should be able to run it. Please provide the relative path to a resource instead. You can move your input file to `src/main/resources` directory and change the path to `src/main/resources/reportToRead.txt`.